### PR TITLE
Allow arbitrary command-line args to be used in Jest

### PR DESCRIPTION
**Pull request by eliperelman**
Addresses #287.

@okonet I tried using this in a project with Jest, but I don't think it's working properly; it just seems to run all the tests. Do you think you would be able to tell if my changes are doing what they should, and potentially offer any advice here?

### CODE REVIEW

File: packages/neutrino-preset-jest/src/index.js [hunk 1]
```diff
@@ -94,8 +94,12 @@ module.exports = (neutrino, opts = {}) => {
     return new Promise((resolve, reject) => {
       const configFile = join(tmpdir(), 'config.json');
       const options = normalizeJestOptions(opts, neutrino, args);
-      const cliOptions = { config: configFile, coverage: args.coverage, watch: args.watch };
+      const cliOptions = merge(
+        omit(['inspect', 'quiet', '_', 'version', 'help', 'use', 'options', '$0', 'files'], args),
+        { config: configFile, coverage: args.coverage, watch: args.watch }
+      );
 
+      cliOptions.findRelatedFiles = cliOptions.findRelatedFiles.split(' ');
```
Comment (okonet) [inline packages/neutrino-preset-jest/src/index.js]: I'm not sure I understand this line. The command lint-staged is calling `jest --findRelatedTests file1.js file2.js file3.js`. Is this aligned with this `split` behaviour?

Comment (okonet) [inline packages/neutrino-preset-jest/src/index.js]: Oh I think I start getting it. Can you point me to the Node API of Jest?

Comment (okonet) [inline packages/neutrino-preset-jest/src/index.js]: So I think it should be something like `jest.runCLI({ findRelatedTests: true, '_': [args._]})`

**Comment by eliperelman** [comment]
@mozilla-neutrino/core-contributors anyone have any cycles to try and continue on this implementation while I'm on PTO?

**Comment by eliperelman** [comment]
Superseded by #313.

**Pull request closed without merging**